docs/user/docs/sessions/troubleshooting.md

```diff
@@ -0,0 +1,31 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+??? question "My Capella crashed. What should I do?"
+    Capella can crash for many different reasons. Many issues are issues with Eclipse Capella itself, so we try to escalate issues there. Bugs can be reported to the Eclipse Capella team directly in the [Github repository](https://github.com/eclipse/capella/issues).
+
+    Administrators can see the logs of all sessions to identify the issues remotely. However, the session owner can also see the events in the UI. In your session, please follow these steps:
+
+    1. Open the Capella search
+    1. Search for `Event logs` and click on the following entry:
```

Comment:
What entry to search for?

Reply:
Do you mean which log entry? That depends on Capella and the issue type, e.g. a session crash.